app/components/auto_tag_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AutoTagComponent < ViewComponent::Base
+  def initialize(text:)
+    colours = {
+      "to do": "yellow",
+      "cannot start yet": "grey",
+      "setup complete": "green",
+      "done": "green",
+    }
+
+    @text = text
+    @colour = colours[text.downcase.to_sym] || "grey"
+  end
+
+  def call
+    render GovukComponent::Tag.new(text: @text, colour: @colour)
+  end
```

Comment:
Does that replace the need for erb template?

Reply:
yes